docs/sources/setup/kubernetes.md

```diff
@@ -266,14 +267,14 @@ Because of the AppArmour restriction, to run Beyla as unprivileged container, yo
 - Set `container.apparmor.security.beta.kubernetes.io/beyla: "unconfined"` in your Kubernetes deployment files.
 - Set a modified AppArmour policy which allows Beyla to perform `mount`.
 
-**Note** Since the `beyla` container does not have the privileges required to mount or un-mount the BPF filesystem, this sample leaves the BPF filesystem mounted on the host, even after the sample is deleted. This samples uses a unique path for each namespace to ensure re-use the same mount if Beyla is re-deployed, but to avoid collisions if multiple instances of Beyla is run in different namespaces. 
+**Note** Since the `beyla` container does not have the privileges required to mount or un-mount the BPF filesystem, this sample leaves the BPF filesystem mounted on the host, even after the sample is deleted. This samples uses a unique path for each namespace to ensure re-use the same mount if Beyla is re-deployed, but to avoid collisions if multiple instances of Beyla is run in different namespaces.
```

Comment:
Just a note for future reference, this will no longer be the case in upcoming (major? cc @grcevski ) releases. It's no longer the case in the 'main' branch, so I believe we can delete this paragraph altogeteher - @grcevski thoughts?

Reply:
Let's leave it for now, we can follow up with a PR to remove all references to the BPF file system in docs, examples etc.